### DISCUSSION TITLE
excavating/feature recording needs its own icon and term

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1056] [General] [answered]
The excavating/feature recording units have a trowel icon on the start page but a treehouse icon in the recording form. They should not have either, they would need something new (and in fact also a different term than feature recording. That's for a new ticket after 9.5:

_Originally posted by @urapadmin in https://github.com/urapadmin/kiosk/issues/674#issuecomment-665775464_

**Comment by urapadmin** [comment]
new term necessary. As for why this recording method exists in the first place:

> Here is our answer as to what the purpose of the excavating/feature recording method is:
> https://github.com/urapadmin/kiosk/issues/789#issuecomment-665675211
> Laurel puts in Wheeler's old legacy excavation (!) in. But since we don't have Wheeler's locus records but rather features (his rooms), we needed to collect features instead of loci. Apart from that we needed the normal excavation recording form, with dayplans, narratives and all. And that is why I used the simple excavating form and extended it to show features instead of loci.
> 
> So much for the history

**Comment by lbestock** [comment]
My head cannot currently think of non-legacy data applications for this, but I don't trust myself. I comment because I am thinking about it, if not productively. Since it is legacy data connected in my head the terms and icons (pith helmet) that suggest themselves to me are in that vein, but wouldn't a pith helmet, funny to me, be problematic in broader application?

**Comment by urapadmin** [comment]
I get Luiza in on this. As the inventor of the treehouse, she is our expert in icons for excavation methods.

Apart from the icon we also need a term, which might even be more important.

**Comment by luizaogs** [comment]
Hardly an expert, but am thinking about it. I do think the pith helmet might get you some raised eyebrows :)

I can't really think of how this might be used for things other than legacy data, either, but I am currently brain dead so take that as you will (but what in the world wouldn't be covered by the other available excavation methods?). IF we cannot think of totally different ways in which this can be used, I suppose framing it more broadly as "integrating records" rather than strictly legacy data might be useful for other projects? In that vein an icon dealing with records rather than digging comes to mind (maybe something like sheets of paper because legacy data is on paper ha), but I'm not sure if that's what you're looking for.

**Comment by urapadmin** [comment]
you guys discuss that out...

**Comment by lbestock** [comment]
I like the idea of paper or an icon of a physical notebook a lot. Definitely better than a pith helmet. But leave it open for a bit longer, please, and we can discuss it when we are a little more human.

**Comment by urapadmin** [comment]
I turn this into a github discussion. But you guys should get to being human enough to get to a conclusion here.

**Comment by lbestock** [comment]
I always forget about the discussions because they do not show up on the testing pane, which is where I hang out most. 
No one here will be surprised that I deal in concretes better than abstracts (nor will it surprise that I cannot and do not want to ignore the chance to lighten a mood). I present two options based on the above discussion to move it into more concrete territory.
![image](https://user-images.githubusercontent.com/49291909/135264299-d1d6fa1b-cc18-4e75-96d7-bab191bd7f0e.png)
A notebook of the sort one used to keep archaeological records in back in the stone ages like 2012.
![image](https://user-images.githubusercontent.com/49291909/135264350-e9906690-d814-4fca-b6e3-d1c53ec28840.png)
A fedora of the type Indy wears. Still poking gentle fun at the past adventures of archaeology, without being quite so problematic or open to misinterpretation as a pith helmet.

Advantages of both are that I drew them, so they're super easy to modify and have no licensing issues.

**Comment by lbestock** [comment]
Terms are even harder, of course. I am trying to think about what this IS. It is really a way to group excavation contexts that are not stratigraphic contexts. Here we call them features, but they are not really features in the same way as other features. If we were talking about data we ourselves collected, we would have two ways of grouping features. First, we would have the survey unit in which they were discovered, and we could group them by tagging them later. How is an excavation/feature recording unit different from a survey unit? In both cases one knows it exists prior to entering any data, one is explicitly dealing with collection of materials without stratigraphic context but with known horizontal boundaries (at least, that is the way we use e/f). The differences are how the data are gathered - excavation vs ground walking. That survey pretty much by definition happens at one time on one day, whereas excavation happens over the course of time (hence we have narratives). And that (at least as we've defined it now) collected material for e/f exists only within the non-stratigraphic excavation contexts themselves - that's really the point - whereas for survey the point is collecting materials from the surface of the whole unit. Why do we need them instead of just having features and grouping the features by tag? We created them because the legacy data for Uronarti had some information recorded that was at the level of these groups. That included narrative in the notebooks, and images of broad areas. Neither tags nor survey units fit the bill entirely, and both missed the fact that the method of collecting that data was excavation. 

Very few current archaeologists excavate unstratigraphically, and I am not interested in our software supporting what I think is bad verging on unethical practice. So: we do not support non-stratigraphic excavation context recording and as such this tool can ONLY be for legacy data entry. 

So what we have here is a unit for legacy excavations that groups excavation contexts that were recognized and recorded as discrete but were not stratigraphic contexts in the old excavations. (If they were stratigraphic contexts, one would input legacy data just like new data.) That's not a term, but it is at least closer to a definition of what the thing is. I find both legacy and excavation to be critical here to the type of unit. Non-stratigraphic is, too, though, or else you would use normal excavation unit recording. I can see calling it an excavation/legacy unit. I cannot see calling it a non-stratigraphic excavation/legacy unit.

**Comment by luizaogs** [comment]
I like both icons! I think choosing what best illustrates the chosen term might be the way to go? "Excavation/legacy [data] unit" makes a lot of sense to me, and if that's the term I lean a bit towards the hat, actually. It might be more immediately connected to legacy than the notebook, since some projects are still stuck in the 2012 stone age (and it is MUCH better than the pith helmet).

**Comment by urapadmin** [comment]
but the real difference between the normal excavation units and the excavation/feature recording units is in fact that the latter is not stratigraphic while the former always is, right? So I can see calling it non-stratigraphic unit :o)

But that's my humble opinion. I still leave it to you guys BUT if you want this to make it into v1 you have to come to terms with it!!!

**Comment by urapadmin** [comment]
@lbestock: And yes, you gave me pngs. I would prefer an svg for the final icon. It scales better.

**Comment by urapadmin** [comment]
what about something like 😱 given how horrible nonstratigraphic excavating is? Or 👎 Or 💩 or 🤷‍♀️ or 🙈  ...

**Comment by luizaogs** [comment]
My final vote is for excavation/legacy unit with the fedora (with maybe an explanation somewhere in the system of what distinguishes this type from regular stratigraphic units, but I don't recall whether one gets a definition when creating a unit). I would vote for the monkey -- you both know I love monkeys, and we already have a treehouse -- but that might get a tad confusing :)

**Comment by lbestock** [comment]
Emojis crash the system just like non-stratigraphic excavation crashes all morally upstanding archaeologists! No monkeys. Monkeys are cute and lovely and no one should expect them to be stratigraphic. (Also, I couldn't see at first that he was covering his eyes. Do I need glasses?) Can't be poop because poop can be excavated very responsibly and then is archaeological gold. So, I think we have to go with the fedora, which means I have to figure out how to export an svg from touchdraw. I know how to get one in, but not how to get one out. And we'll call it excavation/legacy with explanation in the users manual. Non-stratigraphic is more precise but it also isn't really a recognizable term, it is a description, and would lead to stumbling.

**Reply by luizaogs** [reply]
We don't have monkey reaction emojis here?! What an oversight. The important thing I had to tell you is that I am pretty sure you can just convert the png to svg online? Or at least that is what I do.

**Comment by urapadmin** [comment]
that often embedds the bitmap into the svg and does not make it a proper vector. Not good. They are really to different ways of constructing an image. If you have used a vector drawing tool you should use that for exporting directly to a vector format.

**Reply by lbestock** [reply]
Oh I did - it took two seconds to figure it out and it should have sent you the email with the SVG. Did you not get that?

**Comment by urapadmin** [comment]
haven‘t looked at it. This was more a general info for Luiza. Not common knowledge that some svgs look like svgs but are in fact just containing a bitmap.

**Reply by luizaogs** [reply]
Since I didn't even know what a bitmap was ... well I do now!

**Comment by urapadmin** [comment, marked as answer]
all right, one of the discussions came to a conclusion. Yippieh: #1223
